src/backend/parser/diagnostics.ts

```diff
@@ -0,0 +1,20 @@
+import * as FastPlist from 'fast-plist';
+import { TextDecoder } from 'util';
+import { Uri, workspace } from 'vscode';
+import { DiagnosticFile } from '../types';
+
+export async function parseDiagnostics(path: string): Promise<DiagnosticFile> {
+
+    const rawFileContents = await workspace.fs.readFile(Uri.file(path));
+
+    const fileContents = new TextDecoder('utf-8').decode(rawFileContents);
+
+    const parsedContents = FastPlist.parse(fileContents);
```

Comment:
I think there is no benefit to add so many new lines here:
```suggestion
    const rawFileContents = await workspace.fs.readFile(Uri.file(path));
    const fileContents = new TextDecoder('utf-8').decode(rawFileContents);
    const parsedContents = FastPlist.parse(fileContents);
```

Reply:
Fixed, thanks.